resources/assets/js/components/PackageDetail.vue

```diff
@@ -164,3 +164,21 @@ export default {
     },
 };
 </script>
+
+<style scoped>
```

Comment:
Before we merge, we should probably move these to a CSS file, unless there's a reason they should be inside the component? I've never used scoped CSS before, but I wouldn't think to look for these styles here in the future.

Reply:
Good thinking. I pull this snippet from a component in another Tighten project and didn't think about it too much. I moved it to our custom css file and removed some unneeded rules.